lib/contributors/prompt.js

```diff
@@ -18,7 +18,7 @@ var contributionChoices = _.flow(
   })
 );
 
-function getQuestions(options, username, contributions) {
+function getQuestions(options, username, contributions, allContributions) {
```

Comment:
should be named `allContributors` instead (but it can be accessed from `options.contributors`)